packages/web-component/src/lib/helpers/helpers.ts

```diff
@@ -267,6 +275,7 @@ export const handleUrlParams = () => {
     exchangeError,
     redirectAuthCodeChallenge,
     redirectAuthCallbackUrl,
+    redirectAuthBackupCallbackUri,
```

Comment:
not critical, naming wise, I understand that technically this is a URI, but having the `redirectAuthCallbackUrl` (which is also URI_ named with `Url`, I would rename the backup callback also "url" and not "uri" for consistency 

wdyt?

Reply:
This is intentional - the URL is actually an http/https based url (app link / universal link). The backup URI is however a custom scheme URI, used only on Android + Opera at the moment. This is to distinguish between them

Reply:
cool 👍 
as far as I know the fact that there is a different scheme (aka myapp:// instead https://) doesn't affect this terminology between url to uri but if it make sense to you its enough